web/viewer.js

```diff
@@ -16,26 +16,26 @@
 
 'use strict';
 
-var DEFAULT_URL = 'compressed.tracemonkey-pldi-09.pdf';
+let DEFAULT_URL = 'compressed.tracemonkey-pldi-09.pdf';
```

Comment:
Sorry, I really should have let a comment about this as well, but unfortunately we cannot do that :-(
Apparently `CHROME` builds need to mess with it, see https://github.com/mozilla/pdf.js/blob/master/web/viewer.js#L27, and I didn't bother looking into if that can be re-factored somehow.